css/styles.css

```diff
@@ -111,7 +117,8 @@
     width: 40px;
 }
 .ab-column .ab-medium-field {
-    width: 225px;
+    width: 100%;
```

Comment:
since this and the following comment cover the same lines of code, I will answer both in the following 👇